scipion/install/plugin_funcs.py

```diff
@@ -306,24 +303,24 @@ def getInstallenv(self, envArgs=None):
         Environment object with the plugin's binaries."""
         if envArgs is None:
             envArgs = []
-        import script
-        env = script.defineBinaries(envArgs)
+        from .script import defineBinaries
```

Comment:
Why import here? Delaying some dependency in .script ?

Reply:
It was there since the beguining.....the reason was/is that the script.py required python2.7 code and couldn't be imported at the top of sicpion script.

THis part, (installation of libraries) is something to be discussed since scipion-app does not need to install any of them for itself....if we "deprecate" python installation. and rely on conda, python virtual environments and/or snaps.